### PR TITLE
fixed message to Our Robots page, removed discord and team handbook

### PR DESCRIPTION
**Pull request by icai0**

### CODE REVIEW

File: resources/index.html [hunk 1]
```diff
@@ -88,7 +88,7 @@ <h2>Available Resources</h2>
         <div class="resources">
         <br> <a href="Robotics Handbook 2020 - 21.pdf" target="_blank">Robotics Handbook</a>
         <!-- <br> <a href="2019-20 Team Contract.pdf" target="_blank">Team Contract</a> -->
-        <br> <a href="https://discord.gg/UHfrjSt" target="_blank">Discord</a>
+        <!--<br> <a href="https://discord.gg/UHfrjSt" target="_blank">Discord</a>-->
```
Comment (lmaxwell24) [inline resources/index.html:91]: just remove the line

File: README.md [hunk 2]
```diff
@@ -14,7 +14,8 @@ Link to the blog: <https://blog.whsrobotics.org>.
...
 - Chetan Gopal
 - Eric Yamaguchi
-- Landon Bayer :)
 - Ivan Cai
+- Landon Bayer
 - Sean Tao
-- Luke Maxwell
\ No newline at end of file
+- Luke Maxwell
+
```
Comment (lmaxwell24) [inline README.md:21]: unnecessary changes to readme

**Review by lmaxwell24 — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -14,7 +14,8 @@ Link to the blog: <https://blog.whsrobotics.org>.
 - Keith Webber
 - Chetan Gopal
 - Eric Yamaguchi
-- Landon Bayer :)
 - Ivan Cai
+- Landon Bayer
```
Comment (LandonBayer) [inline README.md:18]: Needs smiley :(

**Review by LandonBayer — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (icai0) [reply README.md:18]: no

**Review by lmaxwell24 — APPROVED** [review]

**Review by LandonBayer — APPROVED** [review]
